node/bft/src/bft.rs

```diff
@@ -557,7 +573,10 @@ impl<N: Network> BFT<N> {
                 // Initialize a callback sender and receiver.
                 let (callback_sender, callback_receiver) = oneshot::channel();
                 // Send the subdag and transmissions to consensus.
-                consensus_sender.tx_consensus_subdag.send((subdag, transmissions, callback_sender)).await?;
+                consensus_sender
+                    .tx_consensus_subdag
+                    .send((subdag, transmissions, prior_included_transmissions, aborted_transmissions, callback_sender))
```

Comment:
I wonder if a helper `struct` to hold all these would make sense yet

Reply:
Definitely: 687d56779

There is still a lot of duplication inside of snarkVM's `fn prepare_advance_to_next_quorum_block`. https://github.com/AleoHQ/snarkVM/pull/2304/commits/c349c71d51ac0800f6f355b79d2f36a977c622f4

@raychu86 just a heads up, if and when either of us adds `aborted_solution_ids` we can think of a proper abstraction.